lib/matplotlib/contour.py

```diff
@@ -1183,6 +1183,9 @@ def _contour_level_args(self, z, args):
         if self.filled and len(self.levels) < 2:
             raise ValueError("Filled contours require at least 2 levels.")
 
+        # Sort the contour levels.
+        if len(self.levels) > 1:
+            self.levels = list(sorted(set(self.levels)))
```

Comment:
OK, removed the check immediately below.

Looked into *not* making contours unique.  Its seems OK for `contour`, but `contourf` insists on it, and they gets into triangulations and c-code.  I can imagine that `contourf` would have trouble w/ two equal contours depending on how the patches are drawn, but I didn't look into it too carefully. 

I'd argue the proposed change gives the routines below what they want, but handles a "common" user trip up.  

I've had this happen a lot of times when I'm using every fifth value in an array as a contour interval, but of course the array is going the wrong direction, or is non-monotonic for some reason.